glowfic-dl.py

```diff
@@ -339,7 +477,7 @@ def get_cookies() -> dict[str, str]:
             (name, cookie) = raw.split("=")
             if name != COOKIE_NAME:
                 raise ValueError(
-                    f'cookie file must start with "{COOKIE_NAME}=" (no quotes)'
+                    'cookie file must start with "%s=" (no quotes)' % COOKIE_NAME
```

Comment:
No objection per se but why this change?

Reply:
Stylistic consistency. I used the f-string in my last PR out of habit, when refactoring `get_cookies` out into its own function, but that made it the only string in the whole script to use an f-string rather than the mod operator, and the inconsistency was bugging me, so I changed it to be in line with the rest of the script.